sematic/ui/src/pipelines/PipelineCalculatorView.tsx

```diff
@@ -0,0 +1,44 @@
+import { useEffect, useMemo } from "react";
+import { useParams } from "react-router-dom";
+import Loading from "../components/Loading";
+import { useFetchLatestRuns, usePipelineNavigation } from "../hooks/pipelineHooks";
+import { Alert } from "@mui/material";
+
+/**
+ * This page doesn't do detailed rendering, its main focus is to 
+ * load the latest runs, pick the first run, then redirect to that
+ * run's detail view.
+ * 
+ * @returns JSX.element
+ */
+export default function PipelineCalculatorView() {
```

Comment:
We're trying to deprecate the `Calculator` wording. Can we call this simply `PipelineView`?

Reply:
Done. Now the GitHub review is confused because I exchanged the names of `PipelineView` and `PipelineRunView`. GitHub cannot correlate which file is renamed and is showing diffs incorrectly. 

I intentionally separate the renaming into a different commit. So this shouldn't confuse future developers. For the GitHub problem, we can probably live with it (or if needed, review two commits separately) 

Thanks! Out of curiosity, what has the term `Calculator` evolved into?

Reply:
`Calculator` is an internal implementation detail for `func`, and shouldn't be exposed externally.